survey_runner_jmeter.rb

```diff
@@ -1,13 +1,18 @@
 require 'rubygems'
 require 'ruby-jmeter'
 
-HOST = 'preprod-surveys.eq.ons.digital'
-PROTOCOL = 'https'
-PORT = '443'
-CONNECT_TIMEOUT_MS = '10000'
-RESPONSE_TIMEOUT_MS = '10000'
-USERS = 300
-REPEAT = 1
+# HOST = 'preprod-surveys.eq.ons.digital'
```

Comment:
Do we need the variables in a commented out state. We could have them as defaults ENV.fetch('HOST', 'preprod-surveys.eq.ons.digital')?